src/internal/utils/bucketHelper.ts

```diff
@@ -802,6 +802,13 @@ export function setMeasuresShowOnSecondaryAxis(items: IBucketItem[], value: bool
     }));
 }
 
+export function removeMeasuresShowOnSecondaryAxis(items: IBucketItem[]): IBucketItem[] {
+    return items.map((item: IBucketItem) => ({
+        ...item,
+        [SHOW_ON_SECONDARY_AXIS]: null,
```

Comment:
should it be `false`?

Reply:
i would set it to `null` to mark it as unsupported OR not existed

Reply:
So it should be `undefined`